esp/services/ws_store/ws_storeService.cpp

```diff
@@ -133,25 +135,34 @@ bool CwsstoreEx::onListStores(IEspContext &context, IEspListStoresRequest &req,
         Owned<IPropertyTreeIterator> iter = stores->getElements("Store");
         ForEach(*iter)
         {
+            IPropertyTree * tree = &iter->query();
             Owned<IEspStoreInfo> store = createStoreInfo();
-            store->setOwner(iter->query().queryProp("@createUser"));
-            store->setName(iter->query().queryProp("@name"));
-            store->setCreateTime(iter->query().queryProp("@createTime"));
-            store->setType(iter->query().queryProp("@type"));
-            store->setDescription(iter->query().queryProp("@description"));
+            store->setOwner(tree->queryProp("@createUser"));
+            store->setName(tree->queryProp("@name"));
+            store->setCreateTime(tree->queryProp("@createTime"));
+            store->setType(tree->queryProp("@type"));
+            store->setDescription(tree->queryProp("@description"));
+            store->setMaxValSize(tree->queryProp("@maxValSize"));
```

Comment:
Use DALI_KVSTORE_MAXVALSIZE_ATT?

Reply:
Defined in another lib, not visible here.